src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -356,16 +360,10 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
                 // for each success, write the new AccountId to the Contact.
                 c.AccountId = accountInserts[i].Id;
                 contactsToUpdate.add(c);
-                
-                //Creating affiliation - Functionality Change --> commenting out this code because we 
-                //don't want an affiliation automatically created from a Contact to its parent Account. 
-				//afflsToInsert.add(new Affiliation__c(Contact__c = c.Id, Account__c = accountInserts[i].Id, 
-				//				Primary__c = true, Status__c = 'Current'));
-				
                 i += 1;
             }
         }
-        ADDR_Contact_TDTM.contactsUpdatedAfterInsert = true;
+        ADDR_Contact_TDTM.hasRunAddrTrigger = true;
         update contactsToUpdate;
```

Comment:
This DML statement is not surrounded by any try-catch block. Please add a try-catch block to ensure exceptions are handled properly.